scripts/sync_i18n.js

```diff
@@ -164,6 +165,50 @@ function listLocales() {
     return readdirSync("src/lib/internationalization/locales").map((n) => n.replace(/^(.+)\.\w+$/, "$1"));
 }
 
+function verifyTags() {
```

Comment:
What is this for...? There is a test at [src/test/utils/options/tsdoc-defaults](https://github.com/TypeStrong/typedoc/blob/master/src/test/utils/options/tsdoc-defaults.test.ts) which verifies the consistency between tsdoc.json and TypeDoc's lists of tags.